shim.c

```diff
@@ -81,25 +88,22 @@ int main()
 
   FILE* shim_file;
 
-  if (_wfopen_s(&shim_file, filename, L"r,ccs=UTF-8")) {
+  if ((shim_file = _wfsopen(filename, L"r,ccs=UTF-8", _SH_DENYNO)) == NULL ) {
```

Comment:
Great again, thanks! Though I have the same small nitpick to do as in #11:

```suggestion
  if ((shim_file = _wfsopen(filename, L"r,ccs=UTF-8", _SH_DENYNO)) == NULL) {
```

I don't think the cleanup is necessary as the .exe exits immediately after that, but it's possible sanitizers would complain about this, so thanks again!